<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Stego</class>
 <widget class="QWidget" name="Stego">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>844</width>
    <height>577</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:rgb(255, 255, 255)
</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>0</y>
     <width>851</width>
     <height>581</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QTabWidget" name="encodeDecodeTab">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>-1</y>
      <width>851</width>
      <height>591</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Georgia</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QTabWidget{
background-color: rgb(2,0,36);
background: linear-gradient(90deg, rgba(2,0,36,1) 0%, rgba(134,35,127,1) 36%, rgba(0,0,0,1) 100%);
}</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="encodeTab">
     <property name="accessibleName">
      <string>encodeTab</string>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <attribute name="title">
      <string>Encode</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>70</y>
        <width>281</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Enter the message that</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>100</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>you want to hide :</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="message_Area_Encode">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>170</y>
        <width>251</width>
        <height>81</height>
       </rect>
      </property>
      <property name="accessibleName">
       <string>message_Area</string>
      </property>
      <property name="styleSheet">
       <string notr="true">QTextEdit{
width: 100%;
  height: 150px;
  padding: 12px 20px;
  box-sizing: border-box;
  border: 2px solid #ccc;
  border-radius: 4px;
  background-color: #f8f8f8;
  font-size: 16px;
  resize: none;
 border-radius:10px;
 box-shadow: 0px 0px 2px 2px rgb(0,0,0);
}</string>
      </property>
     </widget>
     <widget class="QPushButton" name="encode_button">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>290</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
        <underline>false</underline>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">  background-color: #4CAF50; /* Green */
  border: none;
  color: white;
  padding: 5px 20px;
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 16px;
 border-radius:10px;
 box-shadow: 0px 0px 2px 2px rgb(0,0,0);</string>
      </property>
      <property name="text">
       <string>Enocde</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="encreption_filed">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>440</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">width: 100%;
  height: 150px;
  box-sizing: border-box;
  border: 2px solid #ccc;
  padding-top: 50px;
  padding-bottom: 50px;
  border-radius: 4px;
  background-color: #f8f8f8;
  font-size: 16px;
  resize: none;
 border-radius:10px;
 box-shadow: 0px 0px 2px 2px rgb(0,0,0);
</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>370</y>
        <width>291</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Your Encreption Key is :</string>
      </property>
     </widget>
     <widget class="QPushButton" name="browse_button_encode">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>440</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
        <underline>false</underline>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{ 
 background-color: #4CAF50; /* Green */
  border: none;
  color: white;
  padding: 5px 20px;
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 16px;
 border-radius:10px;
 box-shadow: 0px 0px 2px 2px rgb(0,0,0);
}
</string>
      </property>
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="path_Filed_Encode">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>440</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
      <property name="accessibleName">
       <string>path_Filed</string>
      </property>
      <property name="styleSheet">
       <string notr="true">width: 100%;
  height: 150px;
  box-sizing: border-box;
  border: 2px solid #ccc;
  padding-top: 50px;
  padding-bottom: 50px;
  border-radius: 4px;
  background-color: #f8f8f8;
  font-size: 16px;
  resize: none;
 border-radius:10px;
 box-shadow: 0px 0px 2px 2px rgb(0,0,0);
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>10</y>
        <width>2</width>
        <height>521</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">  background-color: #4CAF50; /* Green */
  border: none;
  color: white;
  padding: 5px 20px;
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 16px;
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="imagToEncode">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>70</y>
        <width>401</width>
        <height>261</height>
       </rect>
      </property>
      <property name="accessibleName">
       <string>imagToEncode</string>
      </property>
      <property name="styleSheet">
       <string notr="true">
 border: 3px solid #ccc;
 border-radius: 4px;
 border-radius:10px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="decodeTab">
     <attribute name="title">
      <string>Decode</string>
     </attribute>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>100</y>
        <width>291</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>The Hidden Message is :</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="message_Area_Decode">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>150</y>
        <width>251</width>
        <height>81</height>
       </rect>
      </property>
      <property name="accessibleName">
       <string>message_Area</string>
      </property>
      <property name="styleSheet">
       <string notr="true">QTextEdit{
width: 100%;
  height: 150px;
  padding: 12px 20px;
  box-sizing: border-box;
  border: 2px solid #ccc;
  border-radius: 4px;
  background-color: #f8f8f8;
  font-size: 16px;
  resize: none;
 border-radius:10px;
 box-shadow: 0px 0px 2px 2px rgb(0,0,0);
}</string>
      </property>
     </widget>
     <widget class="QPushButton" name="decode_button">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>290</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
        <underline>false</underline>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="accessibleName">
       <string>decode_button</string>
      </property>
      <property name="styleSheet">
       <string notr="true">  background-color: #4CAF50; /* Green */
  border: none;
  color: white;
  padding: 5px 20px;
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 16px;
 border-radius:10px;
 box-shadow: 0px 0px 2px 2px rgb(0,0,0);</string>
      </property>
      <property name="text">
       <string>Denocde</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>380</y>
        <width>301</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Enter Your Dencreption Key :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="decreption_Key">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>440</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">width: 100%;
  height: 150px;
  box-sizing: border-box;
  border: 2px solid #ccc;
  padding-top: 50px;
  padding-bottom: 50px;
  border-radius: 4px;
  background-color: #f8f8f8;
  font-size: 16px;
  resize: none;
 border-radius:10px;
 box-shadow: 0px 0px 2px 2px rgb(0,0,0);
</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>10</y>
        <width>2</width>
        <height>521</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">  background-color: #4CAF50; /* Green */
  border: none;
  color: white;
  padding: 5px 20px;
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 16px;
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="imagToDecode">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>70</y>
        <width>401</width>
        <height>261</height>
       </rect>
      </property>
      <property name="accessibleName">
       <string>imagToEncode</string>
      </property>
      <property name="styleSheet">
       <string notr="true">
 border: 3px solid #ccc;
 border-radius: 4px;
 border-radius:10px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="path_Filed_Decode">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>440</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
      <property name="accessibleName">
       <string>path_Filed</string>
      </property>
      <property name="styleSheet">
       <string notr="true">width: 100%;
  height: 150px;
  box-sizing: border-box;
  border: 2px solid #ccc;
  padding-top: 50px;
  padding-bottom: 50px;
  border-radius: 4px;
  background-color: #f8f8f8;
  font-size: 16px;
  resize: none;
 border-radius:10px;
 box-shadow: 0px 0px 2px 2px rgb(0,0,0);
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="browse_button_decode">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>440</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
        <underline>false</underline>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{ 
 background-color: #4CAF50; /* Green */
  border: none;
  color: white;
  padding: 5px 20px;
  text-align: center;
  text-decoration: none;
  display: inline-block;
  font-size: 16px;
 border-radius:10px;
 box-shadow: 0px 0px 2px 2px rgb(0,0,0);
}
</string>
      </property>
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
